ibis/expr/operations/relations.py

```diff
@@ -585,6 +585,15 @@ def schema(self):
         return backend._get_schema_using_query(self.query)
 
 
+class _UnnestTable(TableNode):
```

Comment:
Can you add a comment about where is it used and why we need it?

Reply:
Yep!

Reply:
Done